api/managementpb/backup/common.proto

```diff
@@ -10,3 +10,10 @@ enum DataModel {
   PHYSICAL = 1;
   LOGICAL = 2;
 }
+
+// BackupMode specifies backup mode.
+enum BackupMode {
+  BACKUP_MODE_INVALID = 0;
+  SNAPSHOT = 1;
+  INCREMENTAL = 2;
```

Comment:
Can we add PITR here as well?
The thing is, SQL engines differentiate between `INCREMENTAL` and `PITR`, so it is by far not the same. If that is true, it'd make sense to have them all here.

Reply:
I thought about it, but then checked `pbm` documentation and discovered that there PITR also called "Incremental backups"

https://www.percona.com/doc/percona-backup-mongodb/point-in-time-recovery.html#incremental-backups

Reply:
On the other hand, it seems that for PG it's different things: https://pgdash.io/blog/postgres-incremental-backup-recovery.html


So yep, you're right. Will add.

Reply:
Done